pennylane/qcut/tapes.py

```diff
@@ -80,16 +80,17 @@ def tape_to_graph(tape: QuantumTape) -> MultiDiGraph:
     order += 1  # pylint: disable=undefined-loop-variable
     for m in tape.measurements:
         obs = getattr(m, "obs", None)
-        if obs is not None and isinstance(obs, Tensor):
+        if obs is not None and isinstance(obs, (Tensor, qml.ops.Prod)):
```

Comment:
```suggestion
        if obs is not None and obs.arithmetic_depth > 0:
```

Potentially a solution.  If we only want *pure* observables here.  Highly unlikely we support sums.

Reply:
This change is causing some issues with `tests/transforms/test_qcut.py::TestTapeToGraph::test_observable_conversion`